cmd/tools/rest/rest.go

```diff
@@ -271,6 +272,70 @@ func FetchData(client *Client, href string, records *[]any, downloadAll bool) er
 	return nil
 }
 
+// FetchRestData used in Rest Collector
+func FetchRestData(client *Client, href string) ([]gjson.Result, error) {
```

Comment:
minor but maybe a better name would be `Fetch` since the package name is `rest` it will be `rest.Fetch` from the callers point of view.